addons/account/models/account_payment.py

```diff
@@ -855,12 +855,22 @@ def create(self, vals_list):
 
         payments = super().create(vals_list)
 
+        # Outstanding account should be set on the payment in community edition to force the generation of journal entries on the payment
+        # This is required because no reconciliation is possible in community, which would prevent the user to reconcile the bank statement with the invoice
+        accounting_installed = 'accountant' in self.env["ir.module.module"]._installed()
```

Comment:
A more logical way of doing this would be to check the value of `_get_invoice_in_payment_state`.

It would also be better for people who use `account` with a custom accounting module (from the community), as they don't rely on a hardcoded name